### ISSUE TITLE
Configuration env vars should work if value is "1" or similar, not just "true"

### ISSUE THREAD

**Issue by jamiew** [git-friendly/git-friendly#74]
Our example in the README explicitly says "set these values to true," and they don't work if you do e.g. `GIT_FRIENDLY_NO_BUNDLE=1` 

I think it might be more obvious if we just checked the presence of this variable at all, or at least had a set of truthy values like "1", "true", "TRUE"

**Comment by sapegin** [comment]
Yeah, make sense. If we're sure we'll never have to unset a variable, then checking for presence should be enough.
